client/ayon_houdini/api/lib.py

```diff
@@ -149,11 +149,23 @@ def validate_fps():
     return True
 
 
-def render_rop(ropnode):
+def render_rop(ropnode, frame_range=tuple()):
```

Comment:
Could this have default value `None`?

Reply:
https://github.com/ynput/ayon-houdini/pull/45/commits/e8922f236c4e267622cd444ca455e0ad2d6bbe6b

Reply:
It could, it had before... but this matches because Houdini's default value and hence avoids the need to do `if frame_range is None: frame_range = tuple()` inside the function.